docs/source/en/tasks/idefics.md

```diff
@@ -357,7 +357,7 @@ for a batch of examples by passing a list of prompts:
 ...     ],
 ... ]
 
->>> inputs = processor(prompts, return_tensors="pt")
+>>> inputs = processor(prompts, return_tensors="pt").to(device)
```

Comment:
"cuda" is used everywhere else in the doc, shouldn't it be the same here?

Reply:
You are right - in all examples above we use "cuda" and should here as well. It is only in the example below that we switch to the device variable. 👍